### PR TITLE
Add zipkin to list of exporting software

### PR DESCRIPTION
**Pull request by devinsba**

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -254,6 +254,7 @@ separate exporters are needed:
    * [VerneMQ](https://github.com/vernemq/vernemq)
    * [Weave Flux](https://github.com/weaveworks/flux)
    * [Xandikos](https://www.xandikos.org/) (**direct**)
+   * [Zipkin](https://zipkin.io/) ([endpoint docs](https://github.com/openzipkin/zipkin/tree/master/zipkin-server#metrics))
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Link directly to the docs, no need for two links.

**Comment by brian-brazil** [comment]
Can you add the DCO?

**Comment by devinsba** [comment]
Done :+1:

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**